substrate/frame/transaction-storage/src/lib.rs

```diff
@@ -349,7 +349,6 @@ pub mod pallet {
 
 	/// Collection of transaction metadata by block number.
 	#[pallet::storage]
-	#[pallet::getter(fn transaction_roots)]
 	pub(super) type Transactions<T: Config> = StorageMap<
```

Comment:
lets make them public, so there is still a way to access them

Reply:
@muharem  Thank you the review. I have implemented the public functions for all the getter storage from line 440-453. Please let me known if there anything else i need to implement.